### PR TITLE
Adds random colours on cable cuffs & crayons, and the ability for flashes to start burnt out

### PR DESCRIPTION
**Pull request by SamHPurp**
More useful as a tool for mappers, so the colour of the cuffs/crayon is random.

Also adds an option for flashes to spawn burnt out. Currently set to 25% of the time. This may be handy for #7497. This is nice because you can't always say that a flash has been used (or emp'd) if found in maint.

No changelog this'll never impact anyone but the mappers, and isn't mapped in anywhere.

### CODE REVIEW

File: code/game/objects/items/crayons.dm [hunk 1]
```diff
@@ -147,6 +147,29 @@
 	colour = "#DA00FF"
 	colourName = "purple"
 
+/obj/item/toy/crayon/random/New()
+	icon_state = pick(list("crayonred", "crayonorange", "crayonyellow", "crayongreen", "crayonblue", "crayonpurple"))
+	switch(icon_state)
```
Comment (taukausanake) [inline code/game/objects/items/crayons.dm]: Why no white in here? You crayon racist.

Semi-related: we don't have black crayons, do we? Totally not intending the race card here, but I _could_ :^)

Off-topic: What would be the opinion of RGB crayons? Maybe random a lot more interesting, but then you'd need a method of getting any color you want. Botany bees mabee?

Comment (SamHPurp) [inline code/game/objects/items/crayons.dm]: White are the mime ones, so I didn't stick it in there.

RGB Crayons would be cool, I suspect someone would do something very cool with it IC art-wise.

Comment (taukausanake) [inline code/game/objects/items/crayons.dm]: I was wondering if white was only mime specific. Does it serve any amazing purpose besides being racially supreme and special only to the mime? I can't think of any other than writing "invisible" text if even. Can you even dye jumpsuits with it?

**Comment by tinfoiltophat** [comment]
mime crayon can change between black and white, whereas the WHITE crayon is a detective thing.

**Comment by Taaketa** [comment]
Make sure flashes in Robotics and Tech storage are excluded please.

**Comment by Spacemanspark** [comment]
Robotics flashes don't matter if they're burnt out.

**Comment by taukausanake** [comment]
Besides I think that's handled by the `/obj/item/device/flash/random` subtype defined with its own new() proc

**Comment by SamHPurp** [comment]
@Taaketa @Spacemanspark @taukausanake Thats not how this works. This added *AN OPTION* for *MAP MAKERS* to *CHOOSE* to have them spawn randomly in either variation.

This effects nothing in the current map in any way shape or form.

**Pull request merged**